InvenTree/config_template.yaml

```diff
@@ -240,6 +240,15 @@ remote_login_header: HTTP_REMOTE_USER
 #   hide_admin_link: true
 #   hide_password_reset: true
 
+# Platform UI options
+# pui_settings:
+#   server_list:
+#     my_server1:
+#       host: https://demo.inventree.org/api/
+#       name: InvenTree Demo
```

Comment:
We need a key for the server items too.

Reply:
A key for what, can you please clarify what you mean with server items

Reply:
A key per listed server. The `HostList` type (frontend) consists of a string as key and a dict with host and name. A key that is reliable between reloads is needed so that the selected host does not change between reloads (when the server list gets appended for example).

Reply:
And what is you proposed change here now? Sorry don't get it.